vmaas-go/base/utils/config.go

```diff
@@ -90,13 +94,30 @@ func initClowderDB() {
 	Cfg.DBPassword = clowder.LoadedConfig.Database.Password
 }
 
+func initEnvDB() {
+	Cfg.DBHost = os.Getenv("POSTGRESQL_HOST")
+	Cfg.DBName = os.Getenv("POSTGRESQL_DB")
+	Cfg.DBPort, _ = strconv.Atoi(Getenv("POSTGRESQL_PORT", "5432"))
+	Cfg.DBSslMode = Getenv("POSTGRESQL_SSL_MODE", "prefer")
+	Cfg.DBSslRootCert = os.Getenv("POSTGRESQL_SSL_ROOT_CERT_PATH")
```

Comment:
why some envs have defaults and others dont?

Reply:
For this one I've set an empty value. Any specific one that you have concern with other than this?

Reply:
it was just a general question why there is the inconsistency, if code using the DBSslRootCert can handle None, then it's fine